packages/solidity-contracts/test/messagesIncomingV3.test.ts

```diff
@@ -917,6 +947,34 @@ describe('FuelMessagePortalV3 - Incoming messages', () => {
       ).to.be.equal(false);
     });
 
+    it('Should not be able to relay message with withdrawal amount exceeding rate limit', async () => {
```

Comment:
I think there is no unit test proving that you can withdraw again after the period has ended

Reply:
there is one in integration test will also add here